keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/EnvironmentController.kt

```diff
@@ -0,0 +1,25 @@
+package com.netflix.spinnaker.keel.rest
+
+import com.netflix.spinnaker.keel.persistence.DeliveryConfigRepository
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/environments"])
+class EnvironmentController(
+  private val deliveryConfigRepository: DeliveryConfigRepository
+) {
+
+  @GetMapping(
```

Comment:
do you need or want a similar api at the delivery config level?

Reply:
I don't know. Do you @erikmunson?

Reply:
I will 100% need one later when we move into the multi-app world, but for now in this moment I really don't know how to make use of that properly in deck so this is sufficient. There are so many fundamental assumptions we have about the app you're in _only_ ever showing things from the one app that I'm going to avoid figuring it out until we know more about what our desired UX is.